src/main/java/com/google/sps/api/calendar/CalendarClientHelper.java

```diff
@@ -94,4 +94,18 @@ public static Event createEventWithSummary(
 
     return event;
   }
+
+  /**
+   * Returns a private event with the given start and end time in the specific time zone
+   * and with a summary and a description.
+   */
+  public static Event createPrivateEventWithSummaryAndDescription(
```

Comment:
It seems as if some of these methods could be extracted into a builder for Calendar Events.

Reply:
Does this mean to create a separate class?

Reply:
yes that was my idea. See https://en.wikipedia.org/wiki/Builder_pattern

Reply:
btw totally ok to do this in a separate refactoring PR